package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@folio/myprofile",
+  "version": "0.1.0",
+  "description": "My profile",
+  "repository": "folio-org/ui-myprofile",
+  "publishConfig": {
+    "registry": "https://repository.folio.org/repository/npm-folio/"
+  },
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=6.0.0"
+  },
+  "stripes": {
+    "type": "settings",
+    "displayName": "ui-myprofile.meta.title",
+    "route": "/myprofile"
+  },
+  "scripts": {
+    "lint": "eslint *.js || true",
+    "test": "echo 'placeholder. no tests implemented'"
+  },
+  "devDependencies": {
+    "@folio/eslint-config-stripes": "^2.0.0",
+    "babel-core": "^6.17.0",
```

Comment:
Aside from perhaps `babel-eslint`, all the other babel dependencies are no longer needed with a ui- module.  Webpack is not needed as well.  Both babel and webpack are taken care of in stripes-core when the app is included in a platform.  Take a look at the boilerplate generated by the Stripes CLI for a new app (`stripes app create` command).  It should have a much more current representation of the minimum ui- module dependencies.

Reply:
Will do.

Reply:
I created the new module and got the following dependences:

  "devDependencies": {
    "babel-eslint": "^8.0.3",
    "eslint": "^4.8.0",
    "@folio/eslint-config-stripes": "^1.1.0"
  },
  "dependencies": {
    "@folio/stripes-components": "^2.0.0",
    "prop-types": "^15.6.0",
    "react-router-dom": "^4.1.1"
  },
  "peerDependencies": {
    "react": "*"
  },

Should I use them in my package.json?

Reply:
The platform provides stripes-core, so unless your app directly depends on something from within stripes-core, you can technically omit it.  Unfortunately I don't think the CLI's ui-app template yet demonstrates use of stripes-connect.

It may be accurate to leave them as peer dependencies.  However, it looks as though stripes-connect is made available via props established in stripes-core.  @MikeTaylor, can you clarify?  For example, I don't see ui-users directly importing from stripes-connect anywhere.  Is this reference a necessity, for clarity, or perhaps leftover from prior work?

Reply:
Guys, please check. I updated MR according to your comments. One question here though. If packages derived from `stripes-core` is it makes sense to provide packages like `react-router-dom` and `prop-types` in package.json?